include/mbedtls/dhm.h

```diff
@@ -301,6 +312,20 @@ size_t mbedtls_dhm_get_bitlen( const mbedtls_dhm_context *ctx );
 size_t mbedtls_dhm_get_len( const mbedtls_dhm_context *ctx );
 
 /**
+ * \brief          This function copies a parameter of a DHM key.
+ *
+ * \param dest     The MPI object to copy the value into. It must be
+ *                 initialized.
+ * \param ctx      The DHM context to query.
+ * \param param    The parameter to copy.
+ *
+ * \return         \c 0 on success.
+ * \return         #MBEDTLS_ERR_DHM_BAD_INPUT_DATA if \p field is invalid.
+ * \return         An \c MBEDTLS_ERR_MPI_XXX error code if the copy fails.
+ */
+int mbedtls_dhm_get_value( mbedtls_mpi *dest,
```

Comment:
Mbed TLS is unfortunately inconsistent. DHM had no functions with both. RSA puts the input first, but bignum, ECP and ECDH put the output first. On the one hand, I think ECDH is the closest thing to DHM so DHM should follow ECDH. On the other hand, I think that if we consider this function to be a getter rather than a copier (which makes sense since it's called `_get_` and not `_copy_`), it is more natural to view the parameters as (context, selector, destination), which go in this order because we put the context first. So ok, I'll change the order.